en/docs/Administer/ProductAdministration/Multitenancy/managing-tenants.md

```diff
@@ -18,37 +18,50 @@ You can add a new tenant in the management console and then view it by following
+
+    !!! info
+        Deleting tenant which is not available in the Management Console UI, can be done through the `RemoteTenantManager` Admin Service. You can invoke these operations using a SOAP client like SOAP UI. Follow the steps below to do the onfigurations using SOAP UI.
 
-        !!! info
-    Other tenant management operations such as activating, deactivating, updating and deleting, which are not available in the Management Console UI, can be done through the `RemoteTenantManager` Admin Service. Y ou can invoke these operations u sing a SOAP client like SOAP UI. Follow the steps below to do the onfigurations using SOAP UI.
 
-    1.  Open the `<API-M_HOME>/repository/conf/carbon.xml` file and set `HideAdminServiceWSDLs` parameter to false.
-    2.  Start SOAP UI client, and import the WSDL `https://localhost:9443/services/RemoteTenantManagerService?wsdl` . This assumes that you are running the SOAP UI client from the same machine as the API Manager instance.
+        1. Open the `<API-M_HOME>/repository/conf/deployment.toml` file and add the following configuration.
+        ``` java
+            [admin_service.wsdl]
+            enable=true
+        ```
+        
+        2.  Start SOAP UI client, and import the WSDL `https://localhost:9443/services/RemoteTenantManagerService?wsdl` . This assumes that you are running the SOAP UI client from the same machine as the API Manager instance.
 
-    3.  Note that there are several operations shown in the SOAP UI after importing the wsdl file:
-        ![]({{base_path}}/assets/attachments/103333432/103333434.png)    4.  Click on each operation to open the request view. For an example, for `activateTenant` operation, you can see the following request view:
-        ![]({{base_path}}/assets/attachments/103333432/103333435.png)
-    5.  You can invoke the `RemoteTenantManager` Admin service and do the tenant activation operation with the corresponding tenant ID. You can perform the other operations via SOAP UI as well. Note that you need to set the admin user credentials from the SOAP UI to invoke tenant admin operations.
+        3.  Note that there are several operations shown in the SOAP UI after importing the wsdl file:
+            ![]({{base_path}}/assets/attachments/103333432/103333434.png)    
+            
+        4.  Click on each operation to open the request view. For an example, for `activateTenant` operation, you can see the following request view:
+            ![]({{base_path}}/assets/attachments/103333432/103333435.png)
+        5.  You can invoke the `RemoteTenantManager` Admin service and do the tenant activation operation with the corresponding tenant ID. You can perform the other operations via SOAP UI as well. Note that you need to set the admin user credentials from the SOAP UI to invoke tenant admin operations.
```

Comment:
A tenant can be activated by passing the corresponding tenant ID  to the tenant activation operation of the `RemoteTenantManager` Admin service.